aws-lc-rs/src/ed25519.rs

```diff
@@ -157,7 +157,7 @@ impl Ed25519KeyPair {
     /// `error::Unspecified` if `rng` cannot provide enough bits or if there's an internal error.
     ///
     /// # FIPS
-    /// FIPS users should not use this method.
+    /// This function should not be used.
```

Comment:
NP: we're not inconsistent with our use of the terms “method” and “function”. This should say “method” — I think any function defined within an `impl` block of a type/struct would be a “method”.

Reply:
I was wrong -- the term "method" also requires the function to take [the `self` param](https://doc.rust-lang.org/book/ch05-03-method-syntax.html#method-syntax):
> Unlike functions, methods are defined within the context of a struct (or an enum or a trait object, ... ), and their first parameter is always `self`, ...